hfc/fabric/channel/channel_eventhub.py

```diff
@@ -318,7 +318,7 @@ def handle_full_tx(self, block, tx_id, er):
                     txFilter = BlockMetadataIndex.Value('TRANSACTIONS_FILTER')
                     txStatusCodes = block['metadata']['metadata'][txFilter]
                     status = TxValidationCode.Name(txStatusCodes[index])
-                    er.onEvent(tx_id, status, block['header']['number'])
+                    er.onEvent(channel_header['tx_id'], status, block['header']['number'])
```

Comment:
The `tx_id` seems to be an input param here from my understanding.

Reply:
> I don't see the change from my fix @xiaomdong , could you review this please :)

Yes，I did not submit your FIX code. 
If you want I submit the Fix code, I will do it.

Reply:
> The `tx_id` seems to be an input param here from my understanding.

Yes, it's an input param.

Reply:
Yes this is the goal @xiaomdong ;)

Reply:
OK，I added the FIX code @GuillaumeCisco

Reply:
> > The `tx_id` seems to be an input param here from my understanding.
> 
> Yes, it's an input param.

Then why are you changing it?

Reply:
Yes it is @dexhunter I've checked with the sdk node.js, the first input should be the txId, not `all`, even if the txid is `empty`.
I don't know if you remember this big discussion about having non empty txId: https://jira.hyperledger.org/browse/FAB-15411?focusedCommentId=61135&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-61135

That's why originally, I've put `all` instead of a potentially empty txid.
But it seems having empty txid even for chaincode `onEvent` callback is not a big deal.

Reply:
ok, but it does not pass the test.

Reply:
Yep, let's fix that, could you look at this @xiaomdong?

Reply:
OK, I will try it @GuillaumeCisco